src/main/java/org/alliancegenome/curation_api/services/validation/dto/Gff3DtoValidator.java

```diff
@@ -243,6 +254,103 @@ public TranscriptGenomicLocationAssociation validateTranscriptLocation(Gff3DTO g
 		return transcriptLocationDAO.persist(locationResponse.getEntity());
 	}
 	
+	@Transactional 
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChBlNDQwZWNkODgwZTEwNjgyEgpjaGVja3N0eWxl -->